### DISCUSSION TITLE
Schema API

### DISCUSSION THREAD

**Discussion by yesoreyeram** [nocodb/nocodb#456] [Q&A]
Are there any REST APIs to get the schema/meta data about the project/table?

**Comment by o1lab** [comment]
For the time being, these arent documented. But they work just fine - could be looked up at network tab.

**Reply by yesoreyeram** [reply]
Hey @o1lab - I love it. Thanks for the quick response. Is this the one you are referring? I see they are working but their path doesn't fall under `/api`. So i doubted `/dashboard` is expected to be consumed outside via GUI?

```js
fetch("http://localhost:8080/dashboard/?q=sqlOp_tableXcModelGet", {
  "headers": {
    "xc-auth": "my-token",
  },
  "body": "{\"api\":\"tableXcModelGet\",\"env\":\"dev\",\"dbAlias\":\"db\",\"project_id\":\"myproject_uOAb\",\"args\":{\"tn\":\"xa1__my_table\"}}",
  "method": "POST"
});
```

**Reply by o1lab** [reply]
Yes, these are the ones. 

Re path : ya we never thought people would love to use meta apis :) - we will think about the path part later.

**Reply by yesoreyeram** [reply]
In long term, having APIs to get list of projects, project schema will be very much helpful.

**Reply by o1lab** [reply]
sure, we will put them out.

**Reply by yesoreyeram** [reply]
That's awesome. I created #457 for tracking purpose 👍🏼
